error from /reports (Top 10)

When going to reports for the top 10 (Offenders, Flapping, Standing), get this error:

2019-12-16 18:31:08,285 flask.app[67]: [ERROR] Exceeded memory limit for $group, but didn't allow external sort. Pass allowDiskUse:true to opt in. request_id=c38a5d3e-6d72-4bd6-b2d9-f56045ba3a52 ip=10.5.0.18
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.7/site-packages/flask_cors/decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/auth/decorators.py", line 74, in wrapped
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/models/metrics.py", line 259, in wrapped
    response = f(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/utils/response.py", line 19, in decorated
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/views/alerts.py", line 457, in get_top10_standing
    top10 = Alert.get_top10_standing(query)
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/models/alert.py", line 524, in get_top10_standing
    return db.get_topn_standing(query, topn=10)
  File "/usr/local/lib/python3.7/site-packages/alerta_server-7.4.1-py3.7.egg/alerta/database/backends/mongodb/base.py", line 715, in get_topn_standing
    responses = self.get_db().alerts.aggregate(pipeline)
  File "/usr/local/lib/python3.7/site-packages/pymongo/collection.py", line 2397, in aggregate
    **kwargs)
  File "/usr/local/lib/python3.7/site-packages/pymongo/collection.py", line 2304, in _aggregate
    client=self.__database.client)
  File "/usr/local/lib/python3.7/site-packages/pymongo/pool.py", line 579, in command
    unacknowledged=unacknowledged)
  File "/usr/local/lib/python3.7/site-packages/pymongo/network.py", line 150, in command
    parse_write_concern_error=parse_write_concern_error)
  File "/usr/local/lib/python3.7/site-packages/pymongo/helpers.py", line 155, in _check_command_response
    raise OperationFailure(msg % errmsg, code, response)
pymongo.errors.OperationFailure: Exceeded memory limit for $group, but didn't allow external sort. Pass allowDiskUse:true to opt in.

--- a/alerta/database/backends/mongodb/base.py
+++ b/alerta/database/backends/mongodb/base.py
@@ -638,7 +638,7 @@ class Backend(Database):
             {'$limit': topn}
         ]
 
-        responses = self.get_db().alerts.aggregate(pipeline)
+        responses = self.get_db().alerts.aggregate(pipeline, allowDiskUse=True)
 
         top = list()
         for response in responses:
@@ -675,7 +675,7 @@ class Backend(Database):
             {'$limit': topn}
         ]
 
-        responses = self.get_db().alerts.aggregate(pipeline)
+        responses = self.get_db().alerts.aggregate(pipeline, allowDiskUse=True)
 
         top = list()
         for response in responses:
@@ -711,7 +711,7 @@ class Backend(Database):
             {'$limit': topn}
         ]
 
-        responses = self.get_db().alerts.aggregate(pipeline)
+        responses = self.get_db().alerts.aggregate(pipeline, allowDiskUse=True)
         top = list()
         for response in responses:
             top.append(

